src/Swarm/Language/Syntax.hs

```diff
@@ -530,6 +544,10 @@ pattern TBind v t1 t2 = SBind v (STerm t1) (STerm t2)
 pattern TDelay :: DelayType -> Term -> Term
 pattern TDelay m t = SDelay m (STerm t)
 
+-- | Match a TDelay without syntax
```

Comment:
This comment needs to be updated.

What is the purpose of `SFuture` / `TFuture`?  It seems like it can never actually result from parsing surface syntax, right?  Is it only for pretty-printing values?  As I think I mentioned before I really don't want to add extra stuff to the AST just to be able to pretty-print values.